lms/djangoapps/verified_track_content/tests/test_models.py

```diff
@@ -158,7 +159,7 @@ def test_cohorting_enabled_missing_verified_cohort(self, error_logger):
         self.assertTrue(VerifiedTrackCohortedCourse.is_verified_track_cohort_enabled(self.course.id))
         self._verify_no_automatic_cohorting()
         self.assertTrue(error_logger.called)
-        self.assertIn("course does not have a verified cohort", error_logger.call_args[0][0])
+        self.assertIn("does not exist", error_logger.call_args[0][0])
```

Comment:
Not a particularly great snippet to look for anymore. Do you mind just looking for the actual string expected "cohort named 'blah' does not exist?